src/content/configuration/experiments.mdx

```diff
@@ -209,6 +210,55 @@ Enable additional in-memory caching of modules which are unchanged and reference
 
 Defaults to the value of [`futureDefaults`](#experimentsfuturedefaults).
 
+### experiments.deferImport
+
+Enable support of the tc39 proposal [Deferring Module Evaluation](https://github.com/tc39/proposal-defer-import-eval).
```

Comment:
Nit, but even though the official proposal name is "deferring module evaluation" I think people know it more as "the `import defer` proposal"